parking_lot_src/ParkingLot/Floors.java

```diff
@@ -0,0 +1,128 @@
+package ParkingLot;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.PriorityQueue;
+import java.util.Vector;
+
+public class Floors {
+    int floorNumber;
+    int maxSlotCount;
+    List<Slots> slotsList = new ArrayList<Slots>();
+    Vector<PriorityQueue<Integer>> slotStore = new Vector<PriorityQueue<Integer>>();
+
+    int countVacant[] = new int[4]; //  1--> TRUCK, 2 --> BIKE, 3 -->CAR
+
+    public Floors(int floorNumber, int maxSlotCount) {
+        this.floorNumber = floorNumber;
+        this.maxSlotCount = maxSlotCount;
+
+        for(int i=1;i<=maxSlotCount;i++) {
```

Comment:
Move these code out side of constructor, constructor should be used just to initialise properties and if required it can be used to call methods, Avoid adding business logic to constructor